content/en/blog/_posts/2024-06-20-pod-failure-policy-ga

```diff
@@ -0,0 +1,124 @@
+end state of a Pod. Each rule describes the match condition, which can be
+based on:
+- container exit codes (using the `onExitCodes` property),
+- Pod conditions (using the `onPodConditions` property).
+
+Each rule specifies an action (`action` property) to take in case of a match.
+The possible values include:
+- `Ignore` - don't count the failure towards the `.backoffLimit`
+- `FailJob` - fail the Job and terminate all running Pods
+- `FailIndex` - fail the index corresponding to the failed Pod (can only be
+   used with Indexed Jobs and [Backoff limit per index](/docs/concepts/workloads/controllers/job/#backoff-limit-per-index))
+- `Count` - the default way of handling a Pod failure - count towards the
+   (`.backoffLimit` or `.backoffLimitPerIndex` if specified)
+
+### Pod conditions
+
+A significant part of the development work was for this feature was to introduce
+the built-in `DisruptionTarget` condition to annotate reasons for Pod failures
+due to disruptions. The condition can have the following values of the `reason`
+field, depending on the scenario:
+
+- `PreemptionByKubeScheduler` - Pod preempted by kube-scheduler
+- `DeletionByTaintManager` - Pod evicted by kube-controller-manager due to taints
+- `EvictionByEvictionAPI` - Pod deleted by Eviction API
+- `DeletionByPodGC` - an orphaned Pod deleted by pod GC
+- `TerminationByKubelet` - Pod terminated due to graceful node shutdown or preemption for critical pods.
+
+Notably, Pod failure policy for Jobs can be used against custom user-supplied
+conditions, either added by webhooks or external controllers.
+
+Users may use them to annotate for:
+- annotating user-specific Pod failures, such as configuration issues,
+- annotating Kubernetes failure scenarios which are currently not yet recognized
+  by any built-in conditions (e.g. admission failures due to `OutOfPods`).
+
+### Example
+
+The Pod failure policy snippet below demonstrates how to use to use it to:
+- ignore any Pod disruption recognized by the built-in DisruptionTarget condition
+- fail the Job in case of configuration issues annotated by the custom `ConfigIssue`
+- fail the Job in case of Pod failure with the reserved exit code 42
+- count all other failures towards the default `backoffLimit` (or
+  `backoffLimitPerIndex` if used)
+
+```yaml
+podFailurePolicy:
+  rules:
+  - action: Ignore
+    onPodConditions:
+    - type: DisruptionTarget
+  - action: FailJob
+    onPodConditions:
+    - type: ConfigIssue
+  - action: FailJob
+    onExitCodes:
+      operator: In
+      values: [ 42 ]
+```
+
+## How can you learn more?
+
+- Read the user-facing documentation for
+  [Pod failure policy](/docs/concepts/workloads/controllers/job/#pod-failure-policy) and
+  [Backoff limit per index](/docs/concepts/workloads/controllers/job/#backoff-limit-per-index)
+- Read the tasks page on
+  [Handling retriable and non-retriable pod failures with Pod failure policy](https://kubernetes.io/docs/tasks/job/pod-failure-policy/)
```

Comment:
I think this will work with just `/docs/tasks/...` looking at other blogs. For example for: https://github.com/kubernetes/website/blob/ce5677940176a09ffed03ca5bd3f09a964683eb5/content/en/blog/_posts/2024-03-12-mid-cycle-1.30.md?plain=1#L111. The generated link works ok in this blog: https://kubernetes.io/blog/2024/03/12/kubernetes-1-30-upcoming-changes/#container-resource-based-pod-autoscaling-kep-1610-https-kep-k8s-io-1610